### PR TITLE
Deprecating use of DisplayType parameter in DisplayCapabilities

### PR DESCRIPTION
**Pull request by askirk**
-Deprecating previous get/setDisplayType
-Adding new get/setDisplayName, returning/setting a String
-Testing for new methods
#534

### CODE REVIEW

File: sdl_android/src/androidTest/java/com/smartdevicelink/test/Test.java [hunk 1]
```diff
@@ -404,6 +404,7 @@
         GENERAL_DIDRESULT_LIST.add(GENERAL_DIDRESULT);
         
         GENERAL_DISPLAYCAPABILITIES.setDisplayType(GENERAL_DISPLAYTYPE);
+		GENERAL_DISPLAYCAPABILITIES.setDisplayName(GENERAL_STRING);
```
Comment (brettywhite) [inline sdl_android/src/androidTest/java/com/smartdevicelink/test/Test.java]: indent is off here

File: sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/DisplayCapabilities.java [hunk 2]
```diff
@@ -56,6 +67,7 @@
  */
 public class DisplayCapabilities extends RPCStruct {
 	public static final String KEY_DISPLAY_TYPE = "displayType";
+    public static final String KEY_DISPLAY_NAME = "displayName";
```
Comment (brettywhite) [inline sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/DisplayCapabilities.java]: indent

File: sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/DisplayCapabilities.java [hunk 3]
```diff
@@ -20,12 +23,20 @@
  * 			<th>SmartDeviceLink Ver. Available</th>
```
Comment (brettywhite) [inline sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/DisplayCapabilities.java]: the formatting of this entire doc thing seems to be off

**Comment by askirk** [comment]
Tested against core

File: sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/DisplayCapabilities.java [hunk 4]
```diff
@@ -22,27 +25,35 @@
...
  * 			<td>The type of display
  *			</td>
  * 			<td>SmartDeviceLink 1.0</td>
  * 		</tr>
  * 		<tr>
+ * 			<td>displayName</td>
+ * 			<td>String</td>
+ * 			<td>The name of the display
+ *			</td>
+ * 			<td>SmartDeviceLink 4.4</td>
```
Comment (joeygrover) [inline sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/DisplayCapabilities.java]: Please change to 4.6

**Pull request merged**